frontend/src/components/FloatingButton.vue

```diff
@@ -18,8 +18,8 @@
 
         <div v-else>
           <div v-for="item in categories" :key="item.title">
-            <v-tooltip left>
-              <template v-slot:activator="{ on, attrs }">
+            <!-- <v-tooltip left>
+              <template #activator="{ on, attrs }"> -->
```

Comment:
Why did you comment this out? If it's not relevant just delete this part.

Reply:
Okay, I will do that.